.github/workflows/extrinsic-ordering-check-from-bin.yml

```diff
@@ -1,29 +1,18 @@
 name: Release - Extrinsic Ordering Check
```

Comment:
We can simply call it "Extrinsic Ordering Check"

Reply:
removed it to simplify the script